app/views/restaurants/_moreInfo.html.erb

```diff
@@ -0,0 +1,25 @@
+<h3>Location</h3>
+
+<ul>
+  <li><%= @restaurant.address %></li>
+</ul>
+
+<h3>Opening Hours</h3>
+<ul>
+  <li>Mon - Fri <span> 0:00 - 0:00 (not in db)</span></li>
```

Comment:
If you want to add something in the view, you need to add it in the database, and then access it through the model: https://github.com/wdi-hk-9/Project-SushiMeAway/blob/master/db/migrate/20151102101421_create_restaurants.rb